Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Hidden_User_Created.yml

```diff
@@ -0,0 +1,3098 @@
+id: Suspicious Hidden User Created_new
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Suspicious Hidden User Created_new
+issilent: true
+description: |-
+  This playbook addresses the following alerts:
+
+  - Suspicious Hidden User Created
+
+  Playbook Stages:
+
+  Triage:
+
+  - Retrieve event information about the created user
+
+  Investigation:
+
+  - Check if the user is local or domain.
+  - For domain users: Retrieve AD attributes, including password expiration.
+  - For local users: Run a Powershell command to get "Password Expires" attribute of the local user.
+  - Get risk level for the affected host.
+  - Search for related Script Engine Activity alerts in the alert.
+
+  Containment:
+
+  - For alerts determined to be true positives, suggest to the analyst to disable the user.
+  - Upon analyst approval: Disable the suspicious user account (domain or local).
```

Comment:
Upon receiving the analyst's approval, disable the suspicious user account (domain or local).